CMakeLists.txt

```diff
@@ -250,8 +259,8 @@ set(CMK_USER_SUFFIX ".user")
 if(CMAKE_SYSTEM_NAME STREQUAL "Darwin")
   set(CHARM_OS "darwin")
   set(CMK_MACOSX 1)
+elseif(CMAKE_SYSTEM_NAME STREQUAL "Windows" OR CMAKE_SYSTEM_NAME STREQUAL "CYGWIN")
   set(CMK_SHARED_SUF "dylib")
-elseif(CMAKE_SYSTEM_NAME STREQUAL "Windows")
```

Comment:
the Windows line jumped above the `dylib` line here

Reply:
Fixed.